test/v1/TokenTestUtils.js

```diff
@@ -0,0 +1,1047 @@
+const util = require("util");
+const abi = require("ethereumjs-abi");
+const _ = require("lodash");
+const BN = require("bn.js");
```

Comment:
I see a bunch of tests were moved. Is BigNumber -> BN the only difference?

Reply:
Diff can be seen here: https://github.com/centrehq/centre-tokens/pull/296/commits/7172076a8f478cd5fa53a0dceb42606dc5eda0ce#diff-dab0d1598c750386404320ffdffcb4c5